hs_core/templates/pages/create-resource.html

```diff
@@ -39,8 +39,13 @@
             <h2>Select a resource type and upload files as needed to create a new resource</h2>
 
             <ul>
-            <li>Files you upload here will be grouped together into a "Resource"</li>
-            <li>File size is limited to 1 GB per file</li>
+            <li>Files you upload here will be grouped together into a "Resource" in HydroShare.</li>
+            <li>File size is limited to 1 GB per file for direct browser upload from your local disk.</li>
+            <li>For file size larger than 1GB, you can create an iRODS account in HydroShare user server

```

Comment:
@pkdash I got your point, but not sure what better term we should use here in a simple informative way to inform users. @rayi113 @dtarb Any suggestions on what better term we should be using here to replace "HydroShare user server" on the resource creation page to inform users?